model/Scout.java

```diff
@@ -1,5 +1,153 @@
 package model;
 
-public class Scout {
-    
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.Enumeration;
+import java.util.Properties;
+import java.util.Vector;
+
+import exception.InvalidPrimaryKeyException;
+
+public class Scout extends EntityBase
+{
+	private static final String myTableName = "Scout";
+
+	protected Properties dependencies;
+
+	// GUI Components
+
+	private String updateStatusMessage = "";
+	
+	public Scout(String scoutId) throws InvalidPrimaryKeyException
+	{
+		super(myTableName);
+
+		setDependencies();
+		persistentState = new Properties();
+		lookupAndStore(scoutId);
```

Comment:
I think the condition you pass should be ("ID = " + scoutId) unless sql infers the ID = part but I'm not sure it does